docs/topics/windows-vhd.md

```diff
@@ -0,0 +1,53 @@
+# Windows AKS Azure marketplace images
+
+## Motivation
+
+The primary motivation for producing AKS specific Windows VHDs is to improve the reliability and deployment time for the configuration phase of Windows nodes configured by aks-engine. 
+By performing expensive configuration options and pre-downloading installation artifacts as part of the VHD build process we can accomplish both of these goals.
+
+Producing and publishing AKS specific Windows VHDs to the Azure marketplace also allow us to get targeted windows patches to customers sooner. 
+Today the WIndows/Windows Server team only publish new images containing the latest B week patches. Several times recently C week patches contain fixes to issues customers are facing running Kubernetes on Windows nodes. This means customers would either need to wait an additional 3-4 weeks to get these fixes or another mechanism to developed these patches to Windows nodes would need to be developed.

```

Comment:
it's probably worth calling out that if a C fix isn't deployed, the wait is for the next month's cumulative release (B) which includes the fix.